pandas/core/algorithms.py

```diff
@@ -651,7 +654,7 @@ def value_counts(
     normalize: bool = False,
     bins=None,
     dropna: bool = True,
-) -> ABCSeries:
+) -> "Series":
```

Comment:
ABCSeries doesn't work as a type, AFAIS: Doing a ``x = pd.value_counts(x); reveal_type(x)`` returns Any, if the return type is ABCSeries.